packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -204,7 +204,12 @@ class MenuContent extends React.Component {
   wrapOnKeyDown(item, index) {
     const { onKeyDown } = item.props;
     return ((event) => {
-      event.preventDefault();
+      const shiftTabClicked = (event.shiftKey && event.nativeEvent.keyCode === MenuUtils.KEYCODES.TAB);
+      const tabClicked = (event.nativeEvent.keyCode === MenuUtils.KEYCODES.TAB);
+      if (!(shiftTabClicked || tabClicked)) {
+        event.preventDefault();
```

Comment:
Is it possible to add a jest test to verify? http://paulsturgess.co.uk/blog/2017/04/29/using-enzyme-to-test-a-react-component-onclick-with-event-preventdefault/